frontend/packages/pipelines-plugin/src/components/catalog/providers/useArtifactHubTasksProvider.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import i18next from 'i18next';
+import { CatalogItem, ExtensionHook } from '@console/dynamic-plugin-sdk';
+import { ResourceIcon, useAccessReview } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { TaskModel } from '../../../models/pipelines';
+import { TektonTaskProviders } from '../../pipelines/const';
+import { useGetArtifactHubTasks } from '../apis/artifactHub';
+import { TektonHubTask } from '../apis/tektonHub';
+import { ApiResult } from '../hooks/useApiResponse';
+
+const normalizeArtifactHubTasks = (tektonHubTasks: any[]): CatalogItem<any>[] => {
```

Comment:
Can we have some type definitions for artifactHubTasks?

```suggestion
const normalizeArtifactHubTasks = (artifactHubTasks: ArtifactHubTask[]): CatalogItem<any>[] => {
```